packages/ui/assets/css/fonts.css

```diff
@@ -0,0 +1,59 @@
+/* cyrillic-ext */
```

Comment:
Why are we adding the fonts to the repo instead of loading them from google fonts?